src/index.html

```diff
@@ -3,11 +3,308 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap" rel="stylesheet">
+
+
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    
+    <header  class="header">
+      <div class="header__content-wrapper">
+        <section class="header__topline-content">
+          <a href="#">
+            <img class="header__logo" src="./images/MyBiKE187x56logo.svg" 
+              alt="myBike Logo">
+          </a>
+          
+          <div class="header__icons-wrapper">
+            <a class="header__phone-svg" href="tel:+1-234-555-55-55">
+            </a>
+            <a class="header__menu-svg" href="#menu"></a>
+          </div>              
+        </section>       
+        <div class="header__slogan">  
+          Take the Streets
+        </div>
+      </div>
+          
+    </header>
+ 
+    <main>
+      <div class="divider"></div>
```

Comment:
no need of divider element , you need to use margin and padding for these

Reply:
Is this considered bad practice to use divider? I can imagine that having less margin and padding within the blocks for positioning, makes blocks more usable for other projects, not?

Reply:
done